docs/specification/README.md

```diff
@@ -296,17 +298,17 @@ A Data End record indicates the end of the data section.
 
 | Bytes | Name | Type | Description |
 | --- | --- | --- | --- |
-| 4 | data_section_crc | int32 | CRC32 of all bytes in the data section. A value of 0 indicates the CRC32 is not available. |
+| 4 | data_section_crc | uint32 | CRC32 of all bytes in the data section. A value of 0 indicates the CRC32 is not available. |
```

Comment:
This was a typo in the current spec. It was suppose to be uint32 which matches the `crc` field in the footer.